src/kbmod/result_list.py

```diff
@@ -456,3 +461,88 @@ def save_to_files(self, res_filepath, out_suffix):
         if np.any(stamps_to_save == None):
             stamps_to_save = np.array([])
         np.save(ospath.join(res_filepath, f"all_ps_{out_suffix}.npy"), stamps_to_save)
+
+
+def load_result_list_from_files(res_filepath, suffix, all_mjd=None):
+    """Create a new ResultList from outputted files.
+
+    Parameters
+    ----------
+    res_filepath : string
+        The directory in which to store the results.
+    suffix : string
+        The suffix to append to the output file name
+    all_mjd : list
+        A list of all the MJD timestamps (optional).
+        If not provided, the function looads from the all_times_ file.
+
+    Returns
+    -------
+    results : ResultList
+       The results stored in the given directory with the correct suffix.
+    """
+    # Load the list of all time stamps unless they were pre-specified.
+    if all_mjd is not None:
+        all_times = all_mjd
+    else:
+        times_file_name = ospath.join(res_filepath, f"all_times_{suffix}.txt")
+        all_times = FileUtils.load_csv_to_list(times_file_name, use_dtype=float)[0].tolist()
+    num_times = len(all_times)
+
+    # Create the ResultList to store the data.
+    results = ResultList(all_times)
+
+    # Load the one required file (trajectories).
+    trajectories = FileUtils.load_results_file_as_trajectories(
+        ospath.join(res_filepath, f"results_{suffix}.txt")
+    )
+
+    # Treat the remaining files as optional. Note that the lightcurve (lc_) can be computed
+    # from psi + phi and times (time_) can be computed from lc_indices, so we do not need
```

Comment:
If they don't need to be loaded, do they need to be written out in the first place?

Reply:
Currently they are used by other analysis scripts. We keep writing them for now so as to be backwards compatible.